components/News/styles.module.scss

```diff
@@ -0,0 +1,72 @@
+.newsTitle {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 1em 0;
+}
+
+.newsContainer {
+  align-items: center;
+  justify-content: center;
+}
+
+.newsLayout {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 66%;
```

Comment:
Well... This is quite the terrible idea. 66% of 320px. Woo. I guess we can come back to it when you take on the `p` tags? I'd encourage you to use some `rem` based size constraint though. Or at least push this limit into a media query.